service/controller/v21/adapter/guest_auto_scaling_group.go

```diff
@@ -61,3 +76,15 @@ func workerCountRatio(workers int, ratio float32) string {
 
 	return strconv.Itoa(rounded)
 }
+
+func minDesiredWorkers(minWorkers, maxWorkers, statusDesiredCapacity int) int {
```

Comment:
I don't follow here. Why is the value not just always `minWorkers`? Can we explain this in the godoc with some examples?

Reply:
Added comment. Maybe that helps?